.github/workflows/test-rc-release.yaml

```diff
@@ -25,8 +27,73 @@ on:  # yamllint disable-line rule:truthy
         default: 'main'
 
 jobs:
+  check-rc-testing-annocement:
+    runs-on: 'ubuntu-20.04'
+    if: github.event_name == 'schedule'
+    steps:
+      - name: Checkout apache-airflow
+        uses: actions/checkout@v3
+        with:
+          repository: 'apache/airflow'
+
+      - name: Parse the latest 100 GitHub issues from apache-airflow to check providers testing annocement
+        id: parse-airflow-gh-issues
+        run: |
+          # The default limit is 30. Set it to 100 for retrieving more issues
+          rc_issue_url=`gh issue list \
+            --json createdAt,title,url \
+            --limit 100 \
+            --jq 'map(
+                    select(
+                      .title |
+                      contains ("Status of testing Providers that were prepared on ")
```

Comment:
@Lee-W Thanks for looking into this. 
I feel this is a bit risky since the value of `Status of testing Providers that were prepared on` might change from ticket to ticket and we will miss the testing of RC. WDYT?

Reply:
I can think of a couple of solutions like sentiment analysis, but I think they are overkill for this situation.

Reply:
Yes, such a check might be risky. I just took a look at the [airflow repo](https://github.com/apache/airflow/issues?q=is%3Aissue+%22Status+of+testing+Providers+that+were+prepared+on%22). There are around 50 issues in this format, so I'd say this is somewhat reliable. Even sentiment analysis might have some risk. Considering this, I think this might be a good enough solution, and we can always trigger manually if we notice this is not triggered.

Reply:
The issue is actually generated from a script, so the title should be fairly predicatable:

https://github.com/apache/airflow/blob/8dc895cbb64e991893705b04e03a6655c35888fe/dev/breeze/src/airflow_breeze/commands/release_candidate_command.py#L248

If this still seems too unstructured, the *testing status* label can also be checked; I think the label is only used for this particular purpose right now.